cinema/serializers.py

```diff
@@ -125,9 +125,9 @@ class TicketSerializer(serializers.ModelSerializer):
     def validate(self, attrs):
         data = super(TicketSerializer, self).validate(attrs=attrs)
         Ticket.validate_ticket(
-            attrs["row"], 
-            attrs["seat"], 
-            attrs["movie_session"].cinema_hall, 
+            attrs["row"],
+            attrs["seat"],
+            attrs["movie_session"].cinema_hall,
```

Comment:
Ensure that the keys 'row', 'seat', and 'movie_session' exist in 'attrs' before accessing them to prevent potential 'KeyError' exceptions. You can use 'attrs.get('key')' or check if the key is in 'attrs' before accessing it.